Source/WebCore/platform/network/curl/CurlProxySettings.h

```diff
@@ -62,6 +62,10 @@ class CurlProxySettings {
     void setAuthMethod(long);
     long authMethod() const { return m_authMethod; }
 
+    using PlatformDataVariant = std::variant<Mode, std::pair<URL, String>>;
+    WEBCORE_EXPORT PlatformDataVariant platformSerializationData() const;
+    WEBCORE_EXPORT static CurlProxySettings makeProxySettings(PlatformDataVariant);
```

Comment:
Should be `PlatformDataVariant&&` or `IPCData&&` really. No reason to copy the strings and refcount churn.